lsp-pyls.el

```diff
@@ -269,6 +269,12 @@ dot."
   :group 'lsp-pyls
   :package-version '(lsp-mode . "6.1"))
 
+(defcustom lsp-pyls-plugins-autopep8-enabled nil
```

Comment:
This should be `t` and yapf disabled by default.
As autopep8 is preferred over yapf after https://github.com/palantir/python-language-server/pull/315
(Pyls readme should be updated)